content/docs/user-guide/gems/reference/aws/aws-core/resource-mapping-files.md

```diff
@@ -53,6 +54,10 @@ The mapping file also supports three **required** global attributes:
 
 This mapping file has a single mapping: **MyKey**, which is a Lambda function called "ExampleLambda". It is deployed in us-west-2 in the AWS account with account ID "123456789123".
 
+{{< important >}}
+Your AWS account ID should not be included in builds running on untrusted clients.
```

Comment:
Builds seems odd in this context. Maybe, We strongly recommend not including AWS account id in resource mapping files in release or other distributed builds. Utilize access patterns like [APIGateway and Custom domain](https://docs.aws.amazon.com/apigateway/latest/developerguide/how-to-custom-domains.html) names to provide alias points to resources.

Not sure if theres a catch all blog we can use, so this may be tricky. But we need to provide help in what to do instead.